### ISSUE TITLE
Incompatible with < iOS 13.3

### ISSUE THREAD

**Issue by jacobcxdev** [jacobcxdev/Iris#1]
`-[CKConversationListController _updateConversationListsAndSortIfEnabled]` does not exist on earlier versions of iOS 13.
- [x] Find another way to update the conversation list
- [x] Add some form of version checking

**Comment by jacobcxdev** [comment]
For < iOS 13.3
```
- (void)_updateFilteredConversationLists;
- (void)_updateNonPlaceholderConverationLists;
```

**Comment by jacobcxdev** [comment]
Fixed in 95ff08997760bb85cbdbb7f4a914ba4474db68c7.
